spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe(Api::V1::KeywordsController) do
```

Comment:
Should I add some tests to assert we return correct response data?

Reply:
yes @Goose97

Reply:
The team usually uses https://github.com/thoughtbot/json_matchers. See this example: https://github.com/nimblehq/the1-api/blob/development/spec/controllers/the1_badge/v2/badges_controller_spec.rb#L16

Reply:
Fixed in 498ef53ece2e7c66aad601d42f9534e2fa532b36